sound/soc/sof/pm.c

```diff
@@ -215,9 +215,13 @@ static int sof_resume(struct device *dev)
 	if (!sdev->ops->resume)
 		return 0;
 
+	/* reset the previous flag of pm process */
+	sdev->dsp_error_pending = false;
+
 	/* power up DSP */
 	ret = snd_sof_dsp_resume(sdev);
 	if (ret < 0) {
+		sdev->dsp_error_pending = true;
```

Comment:
this flag should only be used for firmware errors, so not here

Reply:
ok, I will remove it.